src/components/Feedback/index.css

```diff
@@ -3,7 +3,8 @@
 /** @define Feedback; weak */
 
 .Feedback {
-  box-shadow: inset 0 0 0 0.25rem var(--green);
+  border-radius: 0.25rem;
```

Comment:
Do we want to reserve the rounded corners and drop-shadow look and feel for interactive elements? I was drawn to the feedback box on the homepage and wanted to click it because of the shadow.

Reply:
That's correct, non-interactive panels shouldn't have rounded corners or shadows, have amended.

This is taken from the developing Barnardo's look and feel:
<img width="1298" alt="screen shot 2019-02-25 at 11 23 32" src="https://user-images.githubusercontent.com/6122118/53355733-9485a000-3921-11e9-9bb4-1800fcbd2a2f.png">